.github/workflows/build_pmd.yml

```diff
@@ -0,0 +1,36 @@
+# Has to be the same name as in build in order to require the same check in GitHub
+name: Build
+
+on:
+  push:
+    branches:
```

Comment:
I never combined two types of filters (branches and paths). Do you know that they are evaluated as conjunction or disjunktion? I thought they use a conjunction: The change has to match the branch and the path.

Reply:
I could not find any specific documentation for this case, but my guess would be that you are right. Otherwise, the PMD job should have run for this PR as it is on a branch matching `pr/pmd/**`. I will remove the branch name patterns as looking for changes to the pmd ruleset file should be good enough.

Reply:
Works now for changes to `ruleset.xml`. See [test run](https://github.com/saros-project/saros/runs/709805667).